tests/test_inventory_client.py

```diff
@@ -87,6 +110,19 @@ def test_team_identifier_not_found_error(cli):
     assert exc_info.value.name == 'identifier1337'
 
 
+def test_team_identifier_not_found_error_universe(
+  cli,
+  team_not_in_old_universe,
+  old_universe):
```

Comment:
Indent with 4 spaces. Move `):` to a new line.

Reply:
👍